source/fundamentals/builders/aggregates.txt

```diff
@@ -975,3 +975,39 @@ aggregation stage:
 Learn more about this helper from the
 `searchMeta() API documentation <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/Aggregates.html#searchMeta(com.mongodb.client.model.search.SearchCollector)>`__.
 
+Atlas Vector Search
+-------------------
+
+.. important::
+
+   To learn what versions of MongoDB Atlas support this feature, see
+   (TODO: link atlas-vector-search/vector-search-stage/#limitations after those docs are published)
+
+Use the ``vectorSearch()`` method to create a ``$vectorSearch`` (TODO: link to Server manual once updated)
+pipeline stage that specifies a **semantic search**. A semantic search is
```

Comment:
Thanks for noticing this -- since this is page describes all the methods as "static factory methods that build aggregation pipeline stages", I thought this could be excessive. The other builders also use "pipeline stage" in their descriptions (e.g. "Use the skip() method to create a $skip pipeline stage"), so I think it could be ok to keep consistent.